onnxruntime/core/providers/qnn/builder/opbuilder/resize_op_builder.cc

```diff
@@ -211,27 +211,6 @@ Status ResizeOpBuilder::ValidateOp(QnnModelWrapper& qnn_model_wrapper, const Nod
                 "QNN EP: coordinate transformation mode '", coordinate_mode.c_str(), "' not supported for Resize op.",
                 "Only 'align_corners' and 'half_pixel' are supported.");
 
-  // Check for a valid "nearest_mode" if the mode is "nearest".
-  if (resize_mode == "nearest") {
```

Comment:
See my other comment, This probably shouldn't be removed until we know the correct behavior of QNN's ResizeNearestNeighbor.